crates/edr_provider/src/data.rs

```diff
@@ -1368,10 +1388,49 @@ pub struct BlockDataForTransaction {
     pub transaction_index: u64,
 }
 
+lazy_static! {
+    static ref CONSOLE_ADDRESS: Address = "0x000000000000000000636F6e736F6c652e6c6f67"
+        .parse()
+        .expect("static ok");
+}
+
+#[derive(Debug)]
+struct EvmInspector {
```

Comment:
Can you please move this to a separate file?

Reply:
Done in [ca9a178](https://github.com/NomicFoundation/hardhat/pull/4659/commits/ca9a178e54f2997856fc25f38c9c43e0f11eaab5)